content-repo/extra-docs/articles/microsoft-auth-guide.md

```diff
@@ -77,21 +77,24 @@ To configure a Microsoft integration that uses this authorization flow with a se
 7. Save the instance.
 8. Run the `!<integration command prefix>-generate-login-url` command in the War Room and follow the instructions. For example, for Microsoft Graph User: `!msgraph-user-generate-login-url`.
 
-#### Using National Cloud
-Some of the Cortex XSOAR-Microsoft integrations support the deployment of national clouds through the self-deployed
+### Using National Cloud
+
+- To see which integrations support natively National Clouds,See the [table below.](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#supported-authentication-flows-for-microsoft-integrations) 
+  - For Microsoft Azure integrations select the appropriate cloud using the *Azure Cloud* parameter.
+  - For Microsoft Defender select the appropriate cloud using the *Endpoint Type* parameter.
+  - For using self-deployment option please select the *Custom* option and follow the instructions below.
+
+- Some of the Cortex XSOAR-Microsoft integrations support the deployment of national clouds through the self-deployed
  authorization flow. For more information about Microsoft National Clouds, refer to the [Microsoft documentation](https://docs.microsoft.com/en-us/graph/deployments).
- Currently, the following integration supports national clouds:
- * O365 Outlook Mail (Using Graph API)  
- 
-In order to use a national cloud, change the *URL* parameter to the corresponding address of the national cloud you are using. For example, see [O365 Outlook Mail (Using Graph API) - National Clouds](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-mail#using-national-cloud), for the supported cloud endpoints in this integration.
+ In order to use a national cloud, change the *URL* parameter to the corresponding address of the national cloud you are using.
```

Comment:
maybe we need to change the display name of `azure_ad_endpoint` in "Azure Kubernetes Services" to `Server URL`
```suggestion
 In order to use a national cloud, change the *Server URL* parameter to the corresponding address of the national cloud you are using.
```